### PR TITLE
Update Ana's bio

### PR DESCRIPTION
**Pull request by ananogal**

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -69,9 +69,9 @@
 - id: ana
   name: Ana Nogal
   image: /assets/img/custom/team/ana_nogal.jpg
-  role: Apprentice
-  shortDescription: Ana is an experienced software developer with a passion for the web and mobile technology. She has worked as a developer for the majority of her career ...
-  fullDescription: <p>Ana is an experienced software developer with a passion for the web and mobile technology. She has worked as a developer for the majority of her career and has had the opportunity to work in a wide variety of projects, with various languages and technologies.</p><p>Ana joined Codurance as an apprentice to improve her technical skills and put the values of Software Craftsmanship into practice. She is committed to enhancing both her own knowledge and that of the wider software development community.</p>
+  role: Software Craftsman
```
Comment (alvarogarcia7) [inline _data/team.yml:72]: do you prefer craftsman or craftswoman?

File: _data/team.yml [hunk 2]
```diff
@@ -186,10 +186,10 @@
   name: Alvaro Garcia
   image: /assets/img/custom/team/alvaro_garcia.jpg
   role: Software Craftsman
-  shortDescription: Alvaro is a software developer who firmly believes in the eXtreme Programming methodology. He's embarked on a lifelong learning process. Alvaro has been working in the IT industry for product companies, consulting firms and as a freelancer since 2005. 
+  shortDescription: Alvaro is a software developer who firmly believes in the eXtreme Programming methodology. He's embarked on a lifelong learning process. Alvaro has been working in the IT industry for product companies, consulting firms and as a freelancer since 2005.
```
Comment (alvarogarcia7) [inline _data/team.yml:189]: I think you modified all spaces before the end of line

Comment (jukecraft) [reply _data/team.yml:189]: I think Ana's editor removed all the excess spaces. Yay!

**Review by jukecraft — COMMENTED** [review]
I like the new bio

**Pull request merged**